### PR TITLE
Improve global typography styles

### PR DESCRIPTION
**Pull request by deioo**
## What's Up

Using OpenSans globally works fine for retina displays, but with smaller fonts the legibility is not very good on standard screens / TVs.

### Solution

Use different fonts (Open Sans and Helvetica Neue) for headers and content.

### Other features

* Update font antialiasing settings
* Update default header sizes
* Fix global grid-specific dimensions by updating the rem-base settings

### CODE REVIEW

File: components/_typography.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+body {
```
Comment (eugeniodepalo) [inline components/_typography.scss]: Can we move this file's contents to a new file called `globals/_default.scss`?

Comment (deioo) [inline components/_typography.scss]: I thought of that, but it's a typography-related property and it makes more sense to control it from this stylesheet

Comment (matteodepalo) [inline components/_typography.scss]: I like the typography file, it makes it easier to find typography related vars

**Comment by matteodepalo** [comment]
This looks great but I'm wondering, could we change the base font size to be 16px? At the moment the 14px font size creates some problems with readability, especially on large screens and especially for this kind of elements:


If I remember correctly foundation has 16px as default.

**Comment by deioo** [comment]
16px is a bit too much wasted space for the amount of data we're displaying at once on a page.
But having a font that looks good at smaller sizes - Helvetica _(or at least better than OpenSans)_ is fixing the dimension problem. Also playing with antialiasing for headers and content is also making the readability a lot better.

I've done all the changes on the standard non-retina display and everything looks much much better.

**Comment by matteodepalo** [comment]
I agree that the change in font family and improvement in anti-aliasing are great improvements that needed to be done, but could we try to see what happens when we make the change? I think a screenshot would be helpful. A lot of people have asked to see thing bigger, and they consistently use the `+` button to increase their browser font size. I think marginally this is caused by the font family but not only by that.

File: components/_typography.scss (hunk 1 above)
Comment (matteodepalo) [inline components/_typography.scss]: Should we make a mixin for this?

Comment (deioo) [inline components/_typography.scss]: It's a single property, not common to create mixins for that

Comment (eugeniodepalo) [inline components/_typography.scss]: The thing is, this kind of goes against the concept of components that we have in paint. We added a guard for every global style in the form of `$include-html-paint-component-name`, and inside that we included the mixin declared by the component.

**Comment by matteodepalo** [comment]
:+1:

**Pull request merged**